docs/troubleshooting.rst

```diff
@@ -52,4 +52,22 @@ $ kill -kill [PID]
 cliquet.storage.exceptions.BackendError: OperationalError [Postgres Service]
 ============================================================================
 
-Make sure that postgres server is running properly.
\ No newline at end of file
+Make sure that postgres server is running properly.
+
+
+bind(): No such file or directory [uwsgi error]
+===============================================
+
+Make sure that the repository that you defined for the socket parameter of uwsgi
```

Comment:
Make sure that the path you defined for the socket parameter of the uwsgi configuration exists.